src/index.html

```diff
@@ -11,8 +11,58 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute in the link tag for stylesheets should refer to a `.css` file, not a `.scss` file. SCSS files are preprocessed into CSS, and browsers understand only CSS. Make sure the SCSS files are compiled into CSS and linked here correctly.